src/config.rs

```diff
@@ -0,0 +1,37 @@
+use std::{collections::HashMap, env, fs, path::Path};
+
+pub struct Configuration {
+    pub torii_url: String,
+    pub block_build_step_ms: u64,
```

Comment:
We will add special kind of transaction to set block_build_step. For now I think we can remove it from configs and just hardcode with some value (i.e. 3 secs)

Reply:
let it stay here and we will replace it with a ISI later?

Reply:
then leave a todo for that

Reply:
https://soramitsu.atlassian.net/browse/RH2-35